phpBB/styles/prosilver/theme/buttons.css

```diff
@@ -194,7 +194,7 @@ button::-moz-focus-inner {
 
 
 /* stylelint-disable selector-no-qualifying-type */
-ul.linklist.bulletin > li.small-icon:before {
```

Comment:
this is deprecated code so useless to change `.bulletin` and `.small` were factored out